### PR TITLE
Issue 1 see list of pokemon

### PR DESCRIPTION
**Pull request by Samuel-Cho**
Screen Shot 2021-02-23 at 7 39 10 PM
Screen Shot 2021-02-23 at 7 39 18 PM
Screen Shot 2021-02-23 at 7 39 25 PM

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 26px;
+}
+
+.list-container {
+  position: absolute;
+  padding-left: 15px;
+}
```
Comment (codingcodymiller) [inline css/layout.css]: These are not layout related, they should be in `styles.css`

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  return capitalizedWord;
+}
+
+var kantoList = [];
+var johtoList = [];
+// var caughtList = [];
+var kantoOl = null;
+var johtoOl = null;
+
+var xhrKanto = new XMLHttpRequest();
```
Comment (codingcodymiller) [inline js/main.js]: Move all global variables to the top of your file

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  kantoOl = document.createElement('ol');
+  kantoOl.setAttribute('class', 'pokemon-list kanto-list');
+  for (var kantoIndex = 0; kantoIndex < 151; kantoIndex++) {
+    kantoList.push(capitalize(xhrKanto.response.pokemon_entries[kantoIndex].pokemon_species.name));
+    var kantoLi = createList(kantoList[kantoIndex]);
+    kantoOl.appendChild(kantoLi);
+  }
+  $listContainer.appendChild(kantoOl);
+});
+xhrKanto.send();
```
Comment (codingcodymiller) [inline js/main.js]: This should be in a function

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  for (var johtoIndex = 151; johtoIndex < 251; johtoIndex++) {
+    johtoList.push(capitalize(xhrJohto.response.pokemon_entries[johtoIndex].pokemon_species.name));
+  }
+  for (var j = 0; j < johtoList.length; j++) {
+    var johtoLi = createList(johtoList[j]);
+    johtoOl.appendChild(johtoLi);
+  }
+  $listContainer.appendChild(johtoOl);
+});
+xhrJohto.send();
```
Comment (codingcodymiller) [inline js/main.js]: This should be in a function

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by Samuel-Cho** [comment]
Screen Shot 2021-02-24 at 10 53 02 AM
Screen Shot 2021-02-24 at 10 37 08 AM
Screen Shot 2021-02-24 at 10 47 25 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**